scipy/spatial/distance.py

```diff
@@ -1153,7 +1153,8 @@ def pdist(X, metric='euclidean', p=2, w=None, V=None, VI=None):
     Y : ndarray
         Returns a condensed distance matrix Y.  For
         each :math:`i` and :math:`j` (where :math:`i<j<n`), the
-        metric ``dist(u=X[i], v=X[j])`` is computed and stored in entry ``ij``.
+        metric ``dist(u=X[i], v=X[j])`` is computed and stored 
+        in the :math:`ij` th entry.
```

Comment:
Sorry didn't follow up on this. Shall I add the correct documentation to the ``pdist`` ? Also isn't the same thing applicable to ``cdist``. Please let me know if I am wrong here. Thanks.